Tests/Microsoft.AppCenter.Test.Windows/Ingestion/Http/NetworkStateIngestionTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AppCenter.Ingestion;
+using Microsoft.AppCenter.Ingestion.Http;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.AppCenter.Test.Windows.Ingestion.Http
+{
+    [TestClass]
+    public class NetworkStateIngestionTest : BaseIngestionTest
```

Comment:
Harmonized test class names like: `NetworkStateIngestionTest` to test `NetworkStateIngestion`
Git didn't handle it